src/java/voldemort/store/metadata/MetadataStore.java

```diff
@@ -1168,6 +1168,16 @@ public void initNodeId(int nodeId) {
         }
     }
 
+    public static StoreDefinition getStoreDef(String store, MetadataStore metadataStore) {
+        StoreDefinition def = null;
+        if(SystemStoreConstants.isSystemStore(store)) {
```

Comment:
This function is lifted from here. https://github.com/voldemort/voldemort/blob/master/src/java/voldemort/server/protocol/admin/FetchStreamRequestHandler.java#L143
I did not make the code use the same method as I don't want to test the Stream request handling part.

DataCleanupJob is only scheduled for existing stores, so null is never a possibility.  But since this can be used in the future, I will add the null check.